core/src/main/java/tc/oc/pgm/flag/FlagParser.java

```diff
@@ -218,10 +254,9 @@ public FlagDefinition parseFlag(Element el) throws InvalidXMLException {
     Kit dropKit = factory.getKits().parseKitProperty(el, "drop-kit", null);
     Kit carryKit = factory.getKits().parseKitProperty(el, "carry-kit", null);
     boolean multiCarrier = XMLUtils.parseBoolean(el.getAttribute("shared"), false);
-    boolean sequential = XMLUtils.parseBoolean(el.getAttribute("sequential"), false);
```

Comment:
I agree this change moving the logic to the post is good , but this seems to break the use of sequential on flag elements as it was introduced in #524 ([PGM v0.9](https://github.com/PGMDev/PGM/releases/tag/v0.9)) Any plans to make this new implementation backwards compatible in some way?

Reply:
I did consider backwards compatibility, but ended up deciding it was simply not worth the trouble.
If you want multiple posts you will have to update the map to put them all inside a wrapping post element, where you can define sequential, and other future behaviors for "composite posts" if we decide we want such things. Keeping this support for flag -> multiple posts will simply create us more trouble down the line.
I already talked about it with @calcastor who is the main person here handling map changes and he seems to be fine with this breaking change

Reply:
Sure but mame only handles the maps for OCC(/Stratus?) and not the entire PGM ecosystem(even though its probably most of it)? It would be nice with a warning(Maybe a XML exception?, could refer to this PR in a comment or something) at least if you intend to introduce this breaking change, since the behavior would be unusual compared to the past handling of breaking changes in the XML standard in PGM.

I do agree it is not worth the trouble to properly introduce backwards compatibility in this case though.

Reply:
I've implemented a specific warning message for deprecation